dotcom-rendering/playwright/lib/load-page.ts

```diff
@@ -0,0 +1,33 @@
+import type { Page } from '@playwright/test';
+
+const PORT = 9000;
+const BASE_URL = `http://localhost:${PORT}`;
+
+const loadPage = async (
+	page: Page,
+	path: string,
+	waitUntil: 'load' | 'domcontentloaded' = 'domcontentloaded',
+	region = 'GB',
+): Promise<void> => {
+	await page.addInitScript((regionProvided) => {
+		// force geo region
+		window.localStorage.setItem(
+			'gu.geo.override',
+			JSON.stringify({ value: regionProvided }),
+		);
```

Comment:
Why not use the `@guardian/libs`’s helper, here?

Reply:
Why did you revert the change?

Reply:
Sorry I should have updated this thread. Playwright doesn't have a direct way to use localstorage so you have to use the window of the running browser using [page.addInitScript](https://playwright.dev/docs/api/class-page#page-add-init-script)or [page.evaluate](https://playwright.dev/docs/api/class-page#page-evaluate). However these APIs only allows you to pass serialisable data to the window wrapper so when I tried passing libs.storage it didn't work.

I could look at [page.exposeFunction](https://playwright.dev/docs/api/class-page#page-expose-function) but it seems like a much more convoluted way of doing the same thing for not much benefit.

Either way I think this done as a follow up.